src/modelinstance.cpp

```diff
@@ -82,13 +82,13 @@ Status ModelInstance::loadInputTensors(const ModelConfig& config, const DynamicM
         if (name == ANONYMOUS_INPUT_NAME) {
             continue;
         }
-        if (networkInputs.count(name) == 0) {
+        if (networkInputs.count(name) == 0 && networkInputs.count(config.getRealInputByValue(name)) == 0) {
```

Comment:
What if we do config mapping a=>b & g=>a
And set shape to be: `a: [1,3,224,224]`, should the shape be applied to original `a` or the mapped `a`?

Reply:
I think that currently it would be applied for both, because: 
a=>b: we would find config.getShapes().count(name)
g=>a: we would find config.getShapes().count(config.getMappingInputByKey(name))
If we support such mapping then we would need to check whether input/output is mapped and if thats the case we would look only for mapped_name in config.

Reply:
From user perspective, I think it would be better to apply only to mapped names. As an user I want to map the names because those original ones are long and annoying. Once user maps to understandable names, he is able to forget about original names. @dtrawins what do you think about it?

Reply:
When user defined input/output mapping we should forget about the original name everywhere outside the mapping file.
We shouldn't allow for passing both original name & mapped name as this could lead to issues with transitional mapping a mapped to b, b mapped to c. Less ambiguity the better.